pwa/src/state/enums.ts

```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+
+export enum SEX {
+    male = 'male',
+    female = 'female',
+}
+
+export enum Y_N {
```

Comment:
What is our naming convention for enumerations? All caps? camelCase?

I'm not super partial, but the `_` in `Y_N` feels out of place.

Reply:
Yeah I'm not sure. Would be up for a rename. I prefer uppercase personally cause sometimes they're a nice way to distinguish from typescript interfaces.

`YN` ?

Reply:
Changed to YN